Prac_05/state_names.py

```diff
@@ -0,0 +1,30 @@
+"""
+CP1404/CP5632 Practical
+State names in a dictionary
+File needs reformatting
+"""
+
+# TODO: Reformat this file so the dictionary code follows PEP 8 convention
+CODE_TO_NAME = {"QLD": "Queensland", "NSW": "New South Wales", "NT": "Northern Territory", "WA": "Western Australia",
+                "ACT": "Australian Capital Territory", "VIC": "Victoria", "TAS": "Tasmania"}
+print(CODE_TO_NAME)
+
+state_code = input("Enter short state: ").upper()
+while state_code != "":
+    try:
+        print(state_code, "is", CODE_TO_NAME[state_code])
+
+    except KeyError:
+        print("Invalid short state")
+
+    finally:  # Finally used to prevent infinite looping from occurring
+        state_code = input("Enter short state: ").upper()
+
```

Comment:
Maybe go check the programming patterns page on exceptions based error checking to avoid the infinite looping ?